tests/ui/mir/invalid_pass_name.rs

```diff
@@ -0,0 +1,7 @@
+//@ check-pass
+//@ compile-flags: -Zmir-enable-passes=+ThisPass,-DoesNotExist
+//! Ensure a warning is emitted if unknown pass names are requested
```

Comment:
Suggestion: can you throw in a *known* MIR pass to demonstrate the known pass is not warned on?

Reply:
And also, does `-Zmir-enable-passes=` accept non-`+`/`-`-prefixed pass names?

Reply:
It does not, an error is emitted while parsing the opt